.github/workflows/scripts/css_check.py

```
@@ -0,0 +1,350 @@
+                allowed_css_patterns,
+                violations,
+                correct_css_imports,
+                embedded_css_violations,
+            )
+
+    return CSSCheckResult(
+        violations, correct_css_imports, embedded_css_violations
+    )
+
+
+def validate_directories_input(input_directories):
```

Ruff (0.12.2)

Missing return type annotation for public function `validate_directories_input`

(ANN201)